src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { parseHtml } from 'utils/parse-html';
+
+export type DescriptionProps = {
+  title: string;
+  markup: string;
+  url: string;
+  name: string;
+};
+
+type DescriptionListProps = {
+  items: DescriptionProps[];
+  id: string;
+};
+
+export const DescriptionList = ({ items, id }: DescriptionListProps) => (
+  <div className="cc-description-list">
+    <dl key={id}>
+      {items
+        .filter(({ markup }) => markup !== null )
+        .map(({ title, markup }) => (
+          <>
```

Comment:
You will need to add a key to the fragment since this is the list of mapped items i.e. `<Fragment key={title}>`.

Reply:
This has now been changed! Thx